crates/rpc/rpc-types/src/eth/trace/filter.rs

```diff
@@ -41,6 +41,20 @@ impl TraceFilter {
     }
 }
 
+impl Default for TraceFilter {
```

Comment:
You can derive this since TraceFilterMode implements it too

Reply:
> You can derive this since TraceFilterMode implements it too

Thank you for this !

Reply:
> You can derive this since TraceFilterMode implements it too

Thank you for this !